app/controllers/api/v1/featured_tags_controller.rb

```diff
@@ -6,30 +6,23 @@ class Api::V1::FeaturedTagsController < Api::BaseController
 
   before_action :require_user!
   before_action :set_featured_tags, only: :index
-  before_action :set_featured_tag, except: [:index, :create]
 
   def index
     render json: @featured_tags, each_serializer: REST::FeaturedTagSerializer
   end
 
   def create
-    @featured_tag = current_account.featured_tags.create!(featured_tag_params)
-    ActivityPub::UpdateDistributionWorker.perform_in(3.minutes, current_account.id)
-    render json: @featured_tag, serializer: REST::FeaturedTagSerializer
+    featured_tag = FeaturedTagService.new.call(current_account, featured_tag_params[:name])
+    render json: featured_tag, serializer: REST::FeaturedTagSerializer
   end
 
   def destroy
-    @featured_tag.destroy!
-    ActivityPub::UpdateDistributionWorker.perform_in(3.minutes, current_account.id)
+    UnfeaturedTagWorker.perform_async(current_account.id, params[:id])
```

Comment:
I think you may still want to confirm the record exists before queueing into Sidekiq.

Reply:
Undo set_featured_tag brefore_action and check it again.